src/tile.cpp

```diff
@@ -398,6 +398,12 @@ void Tile::onAddTileItem(Item* item)
 	for (Creature* spectator : spectators) {
 		spectator->onAddTileItem(this, cylinderMapPos);
 	}
+
+	if (!hasFlag(TILESTATE_PROTECTIONZONE)) {
+		if (item->isCleanable()) {
```

Comment:
Merge these ifs, no need to have separate lines

Reply:
done that